CampusAmbassador.html

```diff
@@ -162,15 +162,15 @@
                 </div>
                 <div class="col-md-6 form-group">
                   <label>Phone Number</label>
-                  <input class="form-control" id="Email" name="Phone Number" placeholder="phone number" type="email" required/>
+                  <input class="form-control" id="Email" name="Phone_Number" placeholder="phone number" type="text" required/>

```

Comment:
Please check for the validation of the Phone No.
It should be of type number and max 10 digits for India